cli/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cloudquery/cloudquery/cli
 go 1.19
 
 require (
-	github.com/cloudquery/plugin-sdk v1.33.1
+	github.com/cloudquery/plugin-sdk v1.34.0
```

Comment:
Our CLI is already using this version:
https://github.com/cloudquery/cloudquery/blob/3a25ce24c2b88a3c244c6312949b12e252644651/cli/go.mod#L6

Not sure why we see this diff